repo2docker/buildpacks/conda/freeze.py

```diff
@@ -58,43 +50,32 @@ def freeze(env_file, frozen_file):
                 return
     print(f"Freezing {env_file} -> {frozen_file}")
 
+    # FIXME: conda-lock 0.8 requires {platform} in template
+    # https://github.com/conda-incubator/conda-lock/pull/78
```

Comment:
https://github.com/conda-incubator/conda-lock/pull/78 is merged but this can be updated in a follow-up PR.